lib/action.js

```diff
@@ -40,6 +55,13 @@ module.exports = async (opts) => {
   let response = order;
   if (!response.settled) response = await getOrder(order);
 
+  // if configured to add limit rebuy orders with the sold profits, do so now
+  if (process.env.CPBB_REBUY_MAX && action.funds < 0) { // just sold
```

Comment:
worth documenting that setting CPBB_REBUY_MAX=0 is a way to setup the configs but temporarily disable this feature (if for instance you think we are going into a bear market)